### PR TITLE
Fixes to Contribution Guide, Get Started

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request contains Contribution guide fixes.

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -3861,7 +3862,7 @@
   "/docs/cse/automation-service/automation-service-integration-framework": "/docs/platform-services/automation-service/automation-service-integration-framework",
   "/docs/send-data/collect-from-other-data-sources/kubernetes": "/docs/send-data/kubernetes",
   "/docs/cloud-soar/mssp": "/docs/cloud-soar/legacy/legacy-cloud-soar-mssp",
-  "/docs/cloud-soar/cloud-soar-delivery-2": "/release-notes-csoar/2024/02/06/application-update", 
+  "/docs/cloud-soar/cloud-soar-delivery-2": "/release-notes-csoar/2024/02/06/application-update",
```
Comment (kimsauce) [inline cid-redirects.json:3865]:
```suggestion
  "/docs/cloud-soar/cloud-soar-delivery-2": "/release-notes-csoar/2024/02/06/application-update", 
```

File: cid-redirects.json [hunk 2]
```diff
@@ -3873,4 +3874,4 @@
   "/docs/manage/partitions-data-tiers/searching-data-tiers": "/docs/manage/partitions/data-tiers/searching-data-tiers",
   "/docs/manage/partitions-data-tiers/flex-pricing/estimate-and-actual-scan-data": "/docs/manage/partitions/flex/estimate-scan-data",
   "/docs/manage/partitions/flex/estimate-and-actual-scan-data": "/docs/manage/partitions/flex/estimate-scan-data"
-} 
+}
```
Comment (kimsauce) [inline cid-redirects.json:3877]:
```suggestion
} 
```

File: docs/contributing/create-edit-doc.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
+---
+id: create-edit-doc
+title: Create or Edit a Doc
+description: Learn how to create or edit a doc, write content in markdown, and submit your changes to our GitHub repository.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Iframe from 'react-iframe';
+
+Here's a high-level overview of how to create a new Sumo Logic doc.
```
Comment (kimsauce) [inline docs/contributing/create-edit-doc.md:10]:
```suggestion
Discovered an error in a document? Learn how to submit a fix, along with a comprehensive guide on creating or editing a Sumo Logic document.
```

File: docs/reuse/contributing/preview.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Before proceeding, you'll need to preview your changes.
+
+1. Go to the [Building Locally](https://github.com/SumoLogic/sumologic-documentation#building-locally) section in our README.
+1. Review installation requirements.
+1. Preview your build.
+
+This constructs and deploys a local version of the Sumo Logic Docusaurus site. Our site relies on Docusaurus, a static site generator. It creates your site as basic static HTML, JavaScript, and CSS files.
+
+We exclusively use Yarn for all installations and builds. Avoid using NPM commands for package installations or updates.
```
Comment (jpipkin1) [inline docs/reuse/contributing/preview.md:9]:
```suggestion
We exclusively use [Yarn](https://classic.yarnpkg.com/en/) for all installations and builds. Avoid using NPM commands for package installations or updates.
```

File: docs/contributing/create-edit-doc.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+### Step 4: Submit your request
+
+<Submit/>
+
+## Create a new doc
+
+To submit more extensive edits, such as creating a new doc, we recommend forking our repo, making changes in a new branch, and submitting a PR for review.
+
+Feel free to reach out to us, the Docs Team, to discuss. We're happy to work with you on the project and talk through rewriting content, changing flow, adding a new topic or section, and deprecating content.
```
Comment (jpipkin1) [inline docs/contributing/create-edit-doc.md:94]: How should they reach out to us? Should we include an email address?

File: docs/contributing/index.md [hunk 6]
```diff
@@ -15,37 +15,37 @@ Sumo Logic Documentation is an Open Source project available as a [GitHub reposi
 There are many ways to contribute to Sumo Docs, beyond writing code. Here's how you can get involved.
```
Comment (jpipkin1) [inline docs/contributing/index.md:15]: You still need to update this article to account for the combined create-edit-doc file.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Very nice, @kimsauce! I have a few suggestions.

**Comment by jpipkin1** [comment]
@kimsauce, the build-and-deploy check fails because this PR removes `/docs/contributing/create-doc/`, but you reference it in the CID file. The only solution is to remove the CID change from this PR and submit it after this PR merges.

I run into this issue all the time (most recently in my PR for threat intel GA)!

File: docs/contributing/index.md (hunk 6 above)
Comment (kimsauce) [reply docs/contributing/index.md:15]: thank you - done!

File: docs/contributing/index.md [hunk 7]
```diff
@@ -15,37 +15,37 @@ Sumo Logic Documentation is an Open Source project available as a [GitHub reposi
...
 
 <div className="box-wrapper" markdown="1">
-<div className="box smallbox1 card">
+<div className="box smallbox card">
   <div className="container">
   <a href="/docs/contributing/edit-doc"><img src={useBaseUrl('img/icons/documentation.png')} alt="icon" width="35"/><h4>Submit a Doc Fix</h4></a>
   </div>
 </div>
```
Comment (jpipkin1) [inline docs/contributing/index.md:22]:
```suggestion
```

File: docs/contributing/create-edit-doc.md (hunk 5 above)
Comment (kimsauce) [reply docs/contributing/create-edit-doc.md]: updated!

File: docs/manage/security/access-keys.md [hunk 8]
```diff
@@ -31,29 +29,36 @@ Managing Access Keys (how to).
...
-1. Click **Create Key** to generate the key. 
-1. The window displays the generated Access ID and Access Key. Copy both before clicking **Done**. After you press **Done**, you will not be able to recover the Access ID and Access Key. <br/><img src={useBaseUrl('/img/security/generated-access-key.png')} alt="Access key successfully created" width="600"/>
+1. Click **Create Key** to generate the key. Be sure to copy both the generated Access ID and Access Key before clicking **Done**.<br/><img src={useBaseUrl('/img/security/generated-access-key.png')} alt="Access key successfully created" style={{border: '1px solid gray'}} width="450"/>
+   :::warning
+   After you click **Done**, you will not be able to recover this Access ID and Access Key.
+   :::
+
+### From the Access Keys page
+
+Adminstrators can create access keys under **Administration** > **Security** > **Access Keys** as an alternative to doing it from the [Preferences page](#from-the-preferences-page).
```
Comment (kimsauce) [inline docs/manage/security/access-keys.md:56]:
```suggestion
Administrators can create access keys under **Administration** > **Security** > **Access Keys** as an alternative to doing it from the [Preferences page](#from-the-preferences-page).
```

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**